src/ai/backend/test/cli_integration/conftest.py

```diff
@@ -128,3 +129,8 @@ def new_keypair_options() -> Tuple[KeypairOption, ...]:
             is_active=False, is_admin=False, rate_limit=100000, resource_policy="default"
         ),
     )
+
+
+@pytest.fixture(scope="module")
+def keypair_resource_policy() -> str:
+    return str(uuid.uuid4())
```

Comment:
Currently keypair resource policy does not have UUID-style IDs but just human-readable names as its primary key.
Let's keep this consistent in the test fixtures.

Reply:
Thank you for the feedback! Then I will fix it using faker.